_includes/one-click-payments.md

```diff
@@ -85,6 +87,33 @@ GET {{ authorization_url }} HTTP/1.1
 Authorization: Bearer <AccessToken>
 ```
 
+{% else %}
+
+When the initial purchase is successful, a `paymentToken` is linked to
+the payment. You can return the value by sending a `GET` request towards the
+payment resource with the `payerReference` included.
```

Comment:
This encourages clients to build the URL to the token. Can't the URL be discovered in a previous response, somehow?

```suggestion
When the initial purchase is successful, a `paymentToken` is linked to
the payment. You can return the value by performing a `GET` request towards the
payment resource with the `payerReference` included.
```